migrator/migrations/frozenschema/v75/notification_schedules.go

```diff
@@ -0,0 +1,33 @@
+package schema
+
+import (
+	"reflect"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+)
+
+var (
+	// CreateTableNotificationSchedulesStmt holds the create statement for table `notification_schedules`.
+	CreateTableNotificationSchedulesStmt = &postgres.CreateStmts{
+		GormModel: (*NotificationSchedules)(nil),
+		Children:  []*postgres.CreateStmts{},
+	}
+
+	// NotificationSchedulesSchema is the go schema for table `notification_schedules`.
+	NotificationSchedulesSchema = func() *walker.Schema {
+		schema := walker.Walk(reflect.TypeOf((*storage.NotificationSchedule)(nil)), "notification_schedules")
+		return schema
+	}()
+)
+
+const (
+	// NotificationSchedulesTableName is the name of the table used to store data
+	NotificationSchedulesTableName = "notification_schedules"
```

Comment:
Think this isn't used and can just be removed (unless the build tags are ticking me yet again 🥴).